models/surveys.go

```diff
@@ -454,7 +469,12 @@ func (api *API) createClassifiers(surveyPK int, surveyID, name string, types []s
 
 	// Insert classifier type selector and retrieve its primary key so that we can
 	// use that to associate the classifier types
-	typeSelectorPK, classifierTypeSelectorID, err := api.insertClassifierTypeSelector(name, surveyPK, tx)
+	classifierTypeSelectorID, err := uuid.NewV4()
```

Comment:
Could do single line as above

Reply:
Actually, not this one, as you need the `classifierTypeSelectorID` afterwards :)